src/io/jesi/backpack/random.cljc

```diff
@@ -55,7 +55,7 @@
     (apply str)))
 
 (defn- gen-str [chars size]
-  (apply str (take size (repeatedly #(rand-nth chars)))))
+  (clojure.string/join (repeatedly size (fn* [] (rand-nth chars)))))
```

Comment:
```suggestion
  (str/join (repeatedly size (fn* [] (rand-nth chars)))))
```
import as str

What's the significance of `fn*`?

Reply:
Removed. That was kibit